doc/bitfileDeployment.md

```diff
@@ -0,0 +1,161 @@
+
+After the FPGA bitfile is created, you can use the [FPGA Interface C API](https://www.ni.com/en-us/support/documentation/supplemental/09/introduction-to-the-fpga-interface-c-api.html) to generate header files and use them as an interface between the C/C++ application and the FPGA target.
+
+### 4.1 Generate the C API
+
+- Select the FPGA VI created (under FPGA Target).
+- Right-Click -> "Launch C API Generator...".
+- Select the output directory.
+Leave Prefix Override blank. 
+If you do not specify a custom prefix, the FPGA Interface C API Generator names files and constants based on the name of the FPGA VI from which the application bitfile was compiled.
+- Click `Generate`.
+- Copy the generated files to a selected directory in your C/C++ project.
+
+The generated FPGA Interface C API consists of the following files: a `.h` file, a `.lvbitx` file, `NiFpga.h`, and `NiFpga.c`:
+
+- **Generated .h File**: The .h file is a C header file that contains all the constants required by function calls in your application. 
+- **Generated .lvbitx Bitfile**: This is a version of the original .lvbitx bitfile, renamed to match the prefix of the constants in the .h header file. NiFpga_Open() must find this file in order to ensure the bitstream is downloaded to the FPGA.
+- **NiFpga.h**: This is a C header file. It is identical for all generated C APIs. It declares all the errors, types, constants, and functions needed to write an application. Most of these functions are defined in NiFpga.c.
+- **NiFpga.c**: This is a C source file that you must include in your application. It is identical for all generated C APIs. It defines all the functions your application can call. NiFpga.c loads and unloads the NiFpga library at runtime, and forwards function calls to that library.
+
+Follow [this link](https://www.ni.com/docs/en-US/bundle/fpga-interface-c-api-ref/page/capi/fpgac.html) to read the C API Help documentation.
+
+### 4.2 Deploy Bitfile into CompactRIO
+
+Because of the generated `.lvbitx` is a version of the original bitfile created from the FPGA VI, and this is used by the generated C API interface, this new bitfile is needed to download into the CompactRIO, instead of the original one.
+
+After creating the C/C++ project, including source code and header files, send the compiled file and the new bitfile into the CompactRIO.
+For this, follow the recomended steps:
+
+- ssh to the CompactRIO.
+- Create a new directory to store the .lvbitx and the C/C++ compiled file.
+```
+$ mkdir <reasonable_directory>
+```
+- Copy build file from Linux computer to CompactRIO.
+```
+$ scp <build_file> <user>@<cRIO_ip>:<reasonable_directory>/.
+```
+- Copy the .lvbitx from the Linux computer to cRIO.
+```
+$ scp <NiFpga_<name_of_FPGA_vi>.lvbitx> <user>@<cRIO_ip>:<reasonable_directory>/.
+```
+By default <user> is admin and password in "blank" in CompactRIOs.
```

Comment:
It is noted that the cRIOs in base data center and summit do have the password different from the "blank".

Reply:
Done.